docs/content/config/advanced/auth-ldap.md

```diff
@@ -126,6 +126,62 @@ To enable LDAP over StartTLS (on port 389), you need to set the following enviro
 - SASLAUTHD_LDAP_START_TLS=yes
 ```
 
+## Active Directory Configurations (Tested with Samba4 AD Implementation)
+
+In addition to LDAP explanation above, when Docker Mailserver is intended to be used with Active Directory (or the equivelant implementations like Samba4 AD DC) the following points should be taken into consideration:
+
+- Samba4 Active Directory requires a **secure connection** to the domain controller (DC), either via SSL/TLS (LDAPS) or via StartTLS.
+- The username equivenlant in Active Directory is `sAMAccountName`.
+- `proxyAddresses` can be used to store email aliases of single users. The convention is to prefix the email aliases with `smtp:` (e.g., smtp:some.name@DOMAIN.COM).
+- Active Directory is used typically not only as LDAP Directory storage, but also as a domain controller, i.e., it will do many things including authenticating users. Mixing Linux and Windows clients requires the usage of [RFC2307 attributes](https://wiki.samba.org/index.php/Administer_Unix_Attributes_in_AD_using_samba-tool_and_ldb-tools), namely `uidNumber`, `gidNumber` instead of the typical `uid`. Assigning different owner to email folders can also be done in this approach, nevertheless [there is a bug at the moment in Docker mailserver that overwrites all permissions](https://github.com/docker-mailserver/docker-mailserver/pull/2256?notification_referrer_id=NT_kwDOAOOOe7MyNTcxNjg5OTQxOjE0OTEzMTQ3#pullrequestreview-811816197) when starting the container. So either a manual fix is necessary now, or a temporary workaround to use a hardcoded `ldap:uidNumber` equals to 5000 until this issue is fixed.
+- To deliver the emails to different members of Active Directory **Distribution Group** (in other simple words, mailing lists), editing *ldap-groups.cf* to add `leaf_result_attribute = mail` and `special_result_attribute = member` using a [manually written user-patch](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/override-defaults/user-patches/) is necessary. This can be achieved as simple as this:
```

Comment:
```suggestion
- To deliver the emails to different members of Active Directory **Distribution Group** (similar to mailing lists), use a [`user-patches.sh` script](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/override-defaults/user-patches/) to modify `ldap-groups.cf` so that it includes `leaf_result_attribute = mail` and `special_result_attribute = member`. This can be achieved simply by:
```

> Active Directory **Distribution Group** (similar to mailing lists)

A quick google suggests that AD Distribution Lists are more like the mailing list feature, while Distribution Groups extend that a bit further with shared mailboxes in the group? I'm not sure if this change to say "similar to" is correct but I assume it is slightly better wording of what you had prior.

I re-arranged the wording after that a little, but you need to additionally update the docs link correctly, look at existing docs that link to the docs and see the relative page links that we use, not absolute doc links (_otherwise versioned docs will incorrectly link to edge version docs_).